mllib/src/test/scala/org/apache/spark/mllib/util/MLUtilsSuite.scala

```diff
@@ -161,10 +161,10 @@ class MLUtilsSuite extends SparkFunSuite with MLlibTestSparkContext {
   }
 
   test("appendBias") {
-    val sv = Vectors.sparse(3, Seq((0, 1.0), (2, 3.0)))
+    val sv = Vectors.sparse(4, Seq((0, 1.0), (2, 3.0)))
```

Comment:
Since in the original test, the last element is non-zero, so ``sv.size == sv.values.length``. Just want to prevent this.